### ISSUE TITLE
Ligatures being rendered on inline/block code elements

### ISSUE THREAD

**Issue by layoaster** [squidfunk/mkdocs-material#4381]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

The issue got [fixed](https://github.com/squidfunk/mkdocs-material/pull/326) long time ago but it's happening again. It affects both block & inline code elements.

### Expected behaviour

Have monospaced fonts rendered without ligatures like on github:

```text
<region>-realtime
```

| Druid datasource |
| :--------------- |
| `<region>-realtime` |

### Actual behaviour

Ligatures are being rendered:

![image](https://user-images.githubusercontent.com/4516478/191039641-6342cd2a-e851-4ff8-9a55-7b1bb513b520.png)

### Steps to reproduce

1. Using JetBrains Mono for code elements.
2. Input any pair of characters like `->` or `-<` ligature in a block of code or inline.

### Package versions

- Python: `python --version`: 3.9.13
- MkDocs: `mkdocs --version`: 1.3.1 
- Material: `pip show mkdocs-material | grep -E ^Version`: 8.5.2

### Configuration

```yaml
site_name: "EA"
site_url: null
repo_name: null
repo_url: null
edit_uri: null

theme:
  name: material
  # Colors
  palette:
    - scheme: default
      toggle:
        icon: material/lightbulb-outline
        name: Switch to dark mode
      primary: indigo
      accent: blue
    - scheme: slate
      toggle:
        icon: material/lightbulb
        name: Switch to light mode
      primary: brown
      accent: orange
  # Fonts
  font:
    text: Roboto
    code: JetBrains Mono

  features:
    - navigation.instant
    - navigation.sections
    - navigation.tabs
    - navigation.expand
    - navigation.top
    - search.suggest
    - search.highlight
    - toc.integrate
```

### System information

- Operating system: Linux/Ubuntu
- Browser: Firefox

**Comment by squidfunk** [comment]
Thanks for reporting. Indeed, that's unintended. Fixed in dcab57dd1.

**Comment by squidfunk** [comment]
By the way, funnily, monospace fonts still have no `"liga"` set on `font-feature-settings`:

https://github.com/squidfunk/mkdocs-material/blob/dcab57dd1cced4b77875c1aa1b53467c62709d31/src/assets/stylesheets/main/_typeset.scss#L56

So, effectively, [we changed nothing for the last 5 years](https://github.com/squidfunk/mkdocs-material/blame/dcab57dd1cced4b77875c1aa1b53467c62709d31/src/assets/stylesheets/main/_typeset.scss#L56) 😅

**Comment by squidfunk** [comment]
Released as part of 8.5.3.
